### ISSUE TITLE
Add a bot icon on bot profile pictures

### ISSUE THREAD

**Issue by roipoussiere** [tootsuite/mastodon#8347]
It's important for some people to make the difference between bot accounts and human accounts. The `Bot` tag under the account profile is a good move in this direction, but it could be better to infor users directly on the timeline.

A small bot icon on the user profile picture could be interesting for that.

Something like that (bot icon for this example courtesy picked from [FontAwesome](https://fontawesome.com/icons/robot?style=solid)): 

![capture du 2018-08-21 19-31-30](https://user-images.githubusercontent.com/1665542/44419151-ac4b2880-a57a-11e8-9559-392e9f9f4207.png)

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by Xkeeper0** [comment]
Would it be better to put a tag before the name, perhaps in the same style of the `Bot` tag on profiles? e.g. `[Bot] Thomas Riker`.

I'm suggesting this because it might make it more accessible to screen readers (_"Bot account Thomas Riker posted..."_) but also because someone could get creative and imitate the icon to mask it or otherwise mess with its visibility. This would also remove the need to have it on the profile full view's avatar, since the `Bot` tag is already there.

Basically, something like the tag Discord uses, but at the beginning of the name instead of the end. (Or at the end, if the name can be shortened to display it.)

![image](https://user-images.githubusercontent.com/1077535/44420606-412a3380-a533-11e8-9226-d454be10b0e1.png)

**Comment by DeadSuperHero** [comment]
This definitely feels like a better way to denote bot accounts. Maybe a hybrid of the two approaches might work? I wouldn't mind seeing a small bot icon with a tag next to the display name.

**Comment by nclm** [comment]
I like the “bot” tag but a little robot head is always welcome, so…

![bot](https://user-images.githubusercontent.com/923008/44422348-2ed7e600-a583-11e8-8e2d-71e6e3a73bca.png)

**Comment by Xkeeper0** [comment]
While we're at it, something like this might also be useful:

!["Show posts from bots" as a column configuration option](https://user-images.githubusercontent.com/1077535/44429524-9a529100-a54c-11e8-923c-a547a1c146dd.png)

**Comment by roipoussiere** [comment]
Sure, and it has been recently suggested on #8293. ;)

**Comment by roipoussiere** [comment]
@nclm I admit that I really, really like your suggestion, it's so cute and I love this kind of small details. :D

**Comment by ashleyhull-versent** [comment]
+1 the little robot.

**Comment by Sylvhem** [comment]
I admit I like @nclm proposal better than the original one.

**Comment by roipoussiere** [comment]
> I admit I like @nclm proposal better than the original one.

For me, both suggestions are good and compatible: we can have the bot icon on the profile picture AND replace the "Add user" icon if the account is a bot.

My suggestion is good for user experience, because we can quickly identify which toots in our timeline comes from bots, while @nclm suggestion is a nice and funny detail.

**Comment by MaciekBaron** [comment]
I can work on this today. Will add the icon from _Font Awesome_.

**Comment by roipoussiere** [comment]
Perfect!

Note that this icon is licensed under [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by-sa/4.0/).

**Comment by MaciekBaron** [comment]
I've just realise we are using font awesome version 4.7, because that's the latest version on NPM. Unfortunately the robot icon was introduced in version 5. We might need to use a different one for now.

**Comment by MaciekBaron** [comment]
Found a newer repo, will try using that.

**Comment by MaciekBaron** [comment]
Ok I have this now implemented, just waiting for us to approve the new version of Font Awesome

**Comment by MaciekBaron** [comment]
This is now implemented.
